packages/error-reporting/system-test/testAuthClient.js

```diff
@@ -357,3 +359,69 @@ describe('Expected Behavior', function() {
     });
   });
 });
+
+describe('error-reporting', function() {
+  const TIMESTAMP = Date.now();
+  const BASE_NAME = 'error-reporting-system-test';
+  function buildName(suffix) {
+    return [TIMESTAMP, BASE_NAME, suffix].join('_');
+  }
+
+  const SERVICE_NAME = buildName('service-name');
+  const SERVICE_VERSION = buildName('service-version');
+
+  var errors;
+  var transport;
+  before(function() {
+    errors = require('../src/index.js')({
+      ignoreEnvironmentCheck: true,
+      serviceContext: {
+        service: SERVICE_NAME,
+        version: SERVICE_VERSION
+      }
+    });
+    transport = new ErrorsApiTransport(errors._config, errors._logger);
+  });
+
+  after(function(done) {
+    transport.deleteAllEvents(function(err) {
+      assert.ifError(err);
+      done();
+    });
+  });
+
+  it('Should correctly publish errors', function(done) {
+    this.timeout(TIMEOUT * 2);
```

Comment:
I've added a comment in the code to make it clear why a scaled version of the timeout is used.